### DISCUSSION TITLE
Dziś jest (riposta 3)

### DISCUSSION THREAD

**Discussion by zaba-monika** [MajkiIT/polish-ads-filter#17780] [General] [answered]
> xxcriticxx : https://github.com/MajkiIT/polish-ads-filter/discussions/17774 : @zaba-monika nie dropsy, konczymy ta impreze

- którą sami rozpoczęliście...zabawne
- gdybyś nie zamykał moich dyskusji to bym nie musiała zakładać kolejnych i bym odpisywała w jednej, ale skoro lubisz spamować...tylko potem nie pisz, że to ja spamuję...
- jeśli znowu poleci jakieś mięso w moją stronę, znowu będę ripostowała (nowe dyskusje), ale dodatkowo od razu zgłoszę do administracji jako nękanie
- zamknęłabym tę dyskusję, skoro sprawa zakończona, ale chyba nie ma przycisku zamknij hahaha, no ale w końcu to wersja Beta...hahaha, (chyba, że o czymś nie wiem / coś przegapiłam)

**Comment by xxcriticxx** [comment, marked as answer]
good luck :)
